spec/models/generic_file_spec.rb

```diff
@@ -744,4 +744,18 @@ def paranoid_edit_permissions
       described_class.where_public
     end
   end
+  describe "to_csv" do
+    before do
+      @file.title = ["My Title"]
+      @file.creator = ["Von, Creator"]
+      @file.resource_type = ["Book", "Other"]
+      @file.rights = ["Mine"]
+      @file.mime_type = "application/pdf"
+    end
+    subject { @file.to_csv }
+    it do
```

Comment:
@mjgiarlo This was my effort to make the test run a bit faster.  If you break it the way you suggest then the subject gets loaded twice, and is slower.